tests/v4/test_xgrid.py

```diff
@@ -85,30 +90,6 @@ def test_invalid_lon_lat():
         XGrid(xgcm.Grid(ds, periodic=False))
 
 
-def test_xgrid_ravel_unravel_index():
```

Comment:
Now tested on the BaseGrid instead (pretty much the exact same test - except generalised)